wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrator.swift

```diff
@@ -73,17 +83,16 @@ final class CoreDataMessagingMigrator: CoreDataMessagingMigratorProtocol {
         self.isInMemoryStore = isInMemoryStore
     }
 
-    func requiresMigration(at storeURL: URL, toVersion version: CoreDataMessagingMigrationVersion) -> Bool {
+    func requiresMigration(at storeURL: URL, toVersion version: Version) -> Bool {
         guard let metadata = try? metadataForPersistentStore(at: storeURL) else {
             return false
         }
         return compatibleVersionForStoreMetadata(metadata) != version
     }
 
-    func migrateStore(at storeURL: URL, toVersion version: CoreDataMessagingMigrationVersion) throws {
-        zmLog.safePublic(
-            "migrateStore at: \(SanitizedString(stringLiteral: storeURL.absoluteString)) to version: \(SanitizedString(stringLiteral: version.rawValue))",
-            level: .info
+    func migrateStore(at storeURL: URL, toVersion version: Version) throws {
+        WireLogger.localStorage.info(
+            "migrateStore at: \(SanitizedString(stringLiteral: storeURL.absoluteString)) to version: \(SanitizedString(stringLiteral: version.rawValue))", attributes: .safePublic
         )
 
         try forceWALCheckpointingForStore(at: storeURL)
```

Comment:
WAL is only enabled for the message DB, not the event DB. I'm not sure if this would be problematic.

Reply:
I haven't seen an issue running it